playbooks/common/openshift-master/post_config.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Configure Node Labels
+  hosts: oo_masters_to_config
+  roles:
+    - openshift_label_nodes
+    - openshift_deploy_registry
+    - openshift_deploy_router
```

Comment:
I think I would rather see infra services have their own playbook subdirectory (openshift-infra-services, maybe), rather than lumping them into the master subdirectory. Also, we only need to run these against a single master, not all defined masters (once we support multi-master).